ddtrace/tracer/option.go

```diff
@@ -200,6 +206,9 @@ func newConfig(opts ...StartOption) *config {
 	c.runtimeMetrics = internal.BoolEnv("DD_RUNTIME_METRICS_ENABLED", false)
 	c.debug = internal.BoolEnv("DD_TRACE_DEBUG", false)
 	c.enabled = internal.BoolEnv("DD_TRACE_ENABLED", true)
+	// TODO(fg) set these to true before going GA with this.
```

Comment:
```suggestion
	// TODO(fg): set these to true before going GA with this.
```
Small nit, but we usually do it like this.